### ISSUE TITLE
Preset data is corrupted on-save

### ISSUE THREAD

**Issue by you-win** [you-win/openseeface-gd#14]
1. Preset data breaks face tracking when loaded in
2. Preset data for other models is being displayed, which is a planned feature but should not be happening now

**Comment by you-win** [comment]
Probably to fix this, the `app-config.json` file needs to be reworked.

```
root (app-config.json)
- metadata
  - some key-value pairs
  - anything stored in the last unfinished tab (App Settings)
- presets
  - <preset_name>
    - model
    - pose
    - features
  - ... more presets ...
```

**Comment by you-win** [comment]
should be fixed by #33
